src/Microsoft.ML.Data/Transforms/ConversionsExtensionsCatalog.cs

```diff
@@ -100,8 +100,9 @@ internal static TypeConvertingEstimator ConvertType(this TransformsCatalog.Conve
         /// <example>
         /// <format type="text/markdown">
         /// <![CDATA[
-        ///  [!code-csharp[KeyToValueMappingEstimator](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/ValueMappingStringToKeyType.cs)]
-        /// ]]></format>
+        /// [!code-csharp[MapKeyToValue](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/KeyToValueToKey.cs)]
```

Comment:
>KeyToValueToKey.cs [](start = 101, length = 18)

I do not see this file in the codebase

Reply:
[https://github.com/dotnet/machinelearning/blob/master/docs/samples/Microsoft.ML.Samples/Dynamic/KeyToValueValueToKey.cs](https://github.com/dotnet/machinelearning/blob/master/docs/samples/Microsoft.ML.Samples/Dynamic/KeyToValueValueToKey.cs)

---
In reply to: [271935868](https://github.com/dotnet/machinelearning/pull/3187#discussion_r271935868) [](ancestors = 271935868)

Reply:
updating it. 

---
In reply to: [271959976](https://github.com/dotnet/machinelearning/pull/3187#discussion_r271959976) [](ancestors = 271959976,271935868)